app/main.py

```diff
@@ -1,2 +1,45 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.load_factor = self.capacity * 0.67
```

Comment:
I think it would be more intuitive to multiply by 2 / 3, since this is a more straightforward representation of what you want to achieve here.